src/site/static/eligibility.liquid

```diff
@@ -194,7 +194,8 @@ pageClass: "page-public-assistance"
 
     {% eligPage "household-members" %}
       <p>
-        Enter details for each person living in your household. Include all adults and children who live in your home and share income and household expenses like bills and food.
+        Enter details for each person living in your household. Include all adults and children who live in your home and share income and household expenses like bills and food. 
+        <strong>Please note that for each individual you may only check one option out of being a spouse or dependent.</strong>
```

Comment:
Let's leave the explanatory text out for now; we'll tackle it with #389

Reply:
Removed in latest commit